bin/_utils.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+declare DOCKERHUB_NAMESPACE="fundocker"
+declare IMAGE_NAME_PREFIX="openshift-"
+declare BASE_SERVICE_IMAGE_PATH="docker/images"
+
+
+# Get the Dockerfile path of a service
+#
+# Usage: _get_service_image_path SERVICE
+function _get_service_image_path() {
+    if [[ -z $1 ]]; then
+        echo "get_service_image_path: service name is missing"
+        exit 1
+    fi
+
+    local service=$1
+    local service_image_path="${BASE_SERVICE_IMAGE_PATH}/${service}/Dockerfile"
+
+    echo "$service_image_path"
+}
+
+
+# Check that the Dockerfile path of a service actually exists
+#
+# Usage: _check_service_image_path SERVICE
+function _check_service_image_path() {
+    if [[ -z $1 ]]; then
+        echo "check_service_image_path: service name is missing"
+        exit 1
+    fi
```

Comment:
Dry way : 
You can add this function : 
```
_check_service_name() {
  if [[ -z $2 ]]; then
    echo "$1: service name is missing"
    exit 1
  fi
}
```

And call it like this : 
`_check_service_name _check_service_image_path $*`